docker-compose.yml

```diff
@@ -2,8 +2,6 @@
 # - `z` makes bind mount content shared among multiple containers.
 # - `Z` makes bind mount content private and unshared.
 
-version: "2.4"
-
```

Comment:
https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete

Reply:
@tyranron, do you mind removing the version or should we just ignore the warning?

Reply:
Discussed: should check whether everything's ok with upgrading the compose version here to 4 in a separate PR.